Makefile.ses

```diff
@@ -0,0 +1,14 @@
+NPM ?= docker run -it -v $(pwd)/lambda/:/code -w /code --entrypoint=npm  node:8.10-alpine
+
+## Build lambda package
+build:
+	$(NPM) update
+	sudo chmod -R 777 lambda
+	cd lambda && zip -r ../lambda.zip * && cd ../
+
+## Verify $EMAIL used as gateway for forwarding
+verify_gateway_email:
+	$(call assert-set,REGION)
+	$(call assert-set,EMAIL)
+	aws --region=$(REGION) ses verify-email-identity --email-address $(EMAIL)
```

Comment:
There is no such terraform resources because it required manual step to confirm email.
The workflow is

`sender@email.com` -> `recipient@technical.mail` | `lambda` | `relay@cloudposse.com` -> `destination@email.org`

where 
`recipient@technical.mail` - configured backed with SES by specifying MX record for `technical.mail`
`relay@cloudposse.com` - should be verified email. You have to add it manually and then you will get email from aws with link that finish verification. After that aws will send emails from that address.